.github/workflows/close-release.yaml

```diff
@@ -3,21 +3,89 @@ name: Close Release
 
 on:
   workflow_dispatch:
-
-
+    inputs:
+      next_release_type:
+        type: choice
+        required: true
+        description: 'Next release type to create'
+        options:
+          - ReleaseCandidate
+          - Patch
+          - Minor
+          - Major
+      release_name:
+        type: choice
+        required: true
+        description: 'Name of the release'
+        options:
+          - providence
+          - saint-luis
 concurrency:
   group: close-release
   cancel-in-progress: true
 
 jobs:
   close-release:
     name: Close Release
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-2-core
     steps:
       - name: Checkout hoprnet repository
         uses: actions/checkout@v3
 
-      - name: Choose template
+      - name: Disable sccache (act-only)
+        if: ${{ env.ACT }}
+        run: |
+          echo "SCCACHE_GHA_ENABLED=false" >> $GITHUB_ENV
+          echo "RUSTC_WRAPPER=" >> $GITHUB_ENV
+
+      - name: Sccache Setup
+        if: ${{ !env.ACT }}
+        uses: mozilla-actions/sccache-action@v0.0.3
+        with:
+          version: "v0.4.0"
+
+      - name: Install Nix
+        uses: cachix/install-nix-action@v20
+        with:
+          github_access_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: cachix/cachix-action@v12
+        if: ${{ !env.ACT }}
+        with:
+          name: hoprnet
+          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+        continue-on-error: true
+
+      - name: get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "dir=$(pwd)/.yarn/cache" >> $GITHUB_OUTPUT
+
+      - name: restore yarn cache
+        uses: actions/cache@v3
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: restore cargo cache
+        uses: actions/cache@v3
+        with:
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock', 'rust-toolchain.toml') }}
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+            .cargo/.package-cache/
+            .cargo/bin/
```

Comment:
I don't think `.cargo/bin/` is necessary, because it should be overwritten each time you run this workflow.

Reply:
Also, make sure to cache the nix environment flake dir.

Reply:
I've inherited the entries from other pipelines. That is decision is more up to development. I will remove it in all pipelines.